modules/developer_manual/pages/_partials/webdav_api/responses/insufficient-privileges-overview.adoc

```diff
@@ -0,0 +1,12 @@
+====== Insufficient Privileges
+
+If the user making the request has insufficient privileges to make the request then a status of `HTTP/1.1 401 Unauthorized` will be returned, along with the following XML in the response’s body:
```

Comment:
In general it should be 403, for example if you try to perform a request as regular user that only admins can do.

The 401 is whenever the user cannot login / auth.

Reply:
@PVince81 can you review again please?